### DISCUSSION TITLE
new name?

### DISCUSSION THREAD

**Discussion by jnsbck** [jaxleyverse/jaxley-mech#2] [Polls]
how about `jaxley-ionchannels` or `jaxley-channels` or `jaxley-channel-lib`

**Comment by philippberens** [comment]
I agree with @jnsbck - but it could also contain synapses, receptors... what about jaxley-lib?

**Comment by huangziwei** [comment]
`jaxley-addons`? We also need to consider the scope of this package. `lib` or `addons` are too generic if we are focusing solely on retinal neurons. But I won't oppose the idea if we plan to expand to all types of neurons later, while prioritizing the retina for now.

**Comment by philippberens** [comment]
I agree - addons will do as a working title. We wouldn't mind if someone would implement some cortical channels and provide them as a pull request.

**Comment by michaeldeistler** [comment]
In NEURON these things are called `mechanisms`, so we could also go with something like `jaxley-mech` or `jaxley-mechanisms`.

**Reply by huangziwei** [reply]
I like `jaxley-mech`, especially since it is defined primarily as "a large piloted combat robot" on [Wiktionary](https://en.wiktionary.org/wiki/mech#English). :D
